GiniVision/Classes/Core/GiniPDFDocument.swift

```diff
@@ -32,6 +33,7 @@ final public class GiniPDFDocument: NSObject, GiniVisionDocument {
     init(data: Data) {
         self.data = data
         self.isReviewable = false
+        self.id = UUID().uuidString
```

Comment:
Extensions in Swift cannot have stored properties (there is a workaround but it would be more confusing in this case). This is the main downside of not being able to use generics 😞.

Reply:
I think that UUID is clear enough in this case right? There is no need to abstract it now.